dependencies/common/lockfiles/bionic/renv.lock

```diff
@@ -0,0 +1,1521 @@
+{
+  "R": {
+    "Version": "4.3.0",
```

Comment:
Note that the R version here is ignored during restore; it's only used to notify the user when "loading" an renv project if the version doesn't match the one in the lockfile.